pdr_backend/predictoor/predictoor_agent.py

```diff
@@ -481,22 +482,23 @@ def get_payout(self):
         pending_slots = query_pending_payouts(
             self.ppss.web3_pp.subgraph_url, up_pred_addr
         )
```

Comment:
Is `self.pred_submitter_mgr.predictoor_up_address()` (a) address of a prediction manager contract itself, (b) address of a particular prediction feed?
- The documentation of PredSubmitterMgr::predictoor_up_address() isn't helpful. It says "@return: address -- str, address of upward predictoor contract".
- In general calling something a "predictoor x address" is unhelpful, because so many things are "predictoor". And the main thing that's a predictoor is the person running the bots.

Please change the name of method `predictoor_up_address()`, and `predictoor_up_address()` to make it more precise.

Then, please  `up_pred_addr` to correspond.

Second: there was no call to get / use `predictoor_down_address()` in this method. Was that intentional, or an inadvertent omission? If the latter, it could be a bug?

Reply:
> We might rename it to `pred_submitter_up_address`, does that sound good?

OK. 

And, clarify the docstring for `pred_submitter_up_address()`.

And, same for the "down" version of course (name, and docstring).

> `up_pred_addr`

Just change this to `up_addr`. This is the lowest-complexity option, and we can get away with it since the variable is local and the method is short.

>> [T] there was no call to get / use predictoor_down_address() in this method. Was that intentional, or an inadvertent omission? If the latter, it could be a bug?
> [B] Since both predict on the same slots it's sufficient to query for just one of them

OK. Let's clarify in the code. My recommendation: above the line that would now read `up_addr = self.pred_submitter_mgr.pred_submitter_up_address()`, write a comment such as `# we only need to query in one direction, since both predict on the same slots`.